src/index.ts

```diff
@@ -1,19 +1,80 @@
 import { Probot } from "probot";
 import fs from 'fs';
+import { V1Secret } from '@kubernetes/client-node/dist/gen/model/v1Secret';
```

Comment:
I think you should be able to import this easily by editing the line one below to:

```
import k8s, { V1Secret } from '@kubernetes/client-node';
```

btw, `require` and `import` in TypeScript (in difference to JavaScript) can be used mostly interchangeably (there are some edge cases, but mostly it's the same).

Reply:
ack, will try it.